test/emptytheme/theme.json

```diff
@@ -1,5 +1,5 @@
 {
-	"$schema": "../../schemas/json/theme.json",
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
```

Comment:
I take it these updates to the emptytheme `theme.json` file were mistakenly commited?

Reply:
Sorry, these are unexpected changes. I'll revert these changes before merging.